src/connections/stream_api.rs

```diff
@@ -194,6 +194,7 @@ impl<State> ConnectedStreamApi<State> {
                     dest: 0,       // TODO change this
                     request_id: 0, // TODO change this
                     source: 0,     // TODO change this
+                    bitfield: None,
```

Comment:
Does Default::default() work?
```suggestion
                    ..Default::default()
```
The advantage is that we can next time update protobufs again without changing this code again.

Reply:
Excellent point, done. It has allowed to drop other default fields too and remove `allow(deprecated)`, nice!